app/src/main/res/layout/topic_lessons_story_summary.xml

```diff
@@ -156,7 +156,6 @@
           android:id="@+id/chapter_recycler_view"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:layout_marginStart="5dp"
```

Comment:
Please note that this line is removed , but github is showing it exists. Please view the whole file and you will see it is fixed.